src/LHScriptX/Lexer.h

```diff
@@ -121,12 +122,11 @@ class Token
 	Type type_;
 	union
 	{
-		std::string* identifierValue;
-		std::string* stringValue;
 		int integerValue;
 		float floatValue;
 		Operator op;
 	} u_;
+	std::string_view s_;
```

Comment:
This really doesn't seem right, should it not be a std::string

Reply:
Who owns the string it's viewing? The Lexer? If it's intended and known let's stick a comment on it saying exactly what it's viewing.

Reply:
Oh, I thought the string_view were a bit like shared_ptr, with the string allocated in the heap. But it actually is more like a raw pointer which could become dangling if the source string is missing?

In this case, maybe I should stick to the first two commits of this MR and forget about the two lasts? We'll have more string copies, but string ownership mill be clearer.

Reply:
Reverted to string.